### DISCUSSION TITLE
Run a blog server like Ghost or Forem install for the PLN

### DISCUSSION THREAD

**Discussion by bmann** [memser-spaceport/pl-network-hub#16] [Ideas]
To make it easy for any individual in the PLN to use open source and DWeb friendly tools instead of Medium or Substack, we could run a big Ghost (https://ghost.org) or Forem (https://www.forem.com/self-host) instance.

We'd pick a domain like https://plnetwork.blog/ and anyone in the network could have an account.

We could use Ghost like @fission-codes -- as a headless backend with a nice editing interface, and then when you publish, the blog posts can be published to IPFS.

Experimenting with using FIL accounts to login might be one way to restrict this to PLN members.

**Comment by bmann** [comment]
There are some other systems -- like WriteFreely https://writefreely.org/ -- that are ActivityPub enabled that could be an even better fit. I'd still want to add IPFS publishing to it, which could also be part of existing efforts to add IPFS to Mastodon https://opencollective.com/fission/contribute/ipfs-media-support-for-mastodon-48315

**Comment by pandeyanujk** [comment]
This is one possible path based on [this discussion](https://github.com/orgs/memser-spaceport/discussions/15). I think having a CMS based Dsite for blogging/marketing would be good. There is also a widely used tool within PLN to consider like Tina CMS ([Microgen sites](https://github.com/MicrogenSite/microgen)) that publish to IPFS already.

**Reply by bmann** [reply]
for this thread, I’m suggesting a single install for 100s or 1000s of people across the PLN to make it very easy for them to blog -> IPFS.

regardless of solution, do you think this IPFS blog Server is a good idea?

I wish Tina was fully OSS — but it’s not. And, it doesn’t publish to IPFS: it’s used to edit static sites which at PL usually use Fleek to publish to IPFS. Probably a good tool for the marketing best practices thread here https://github.com/orgs/memser-spaceport/discussions/15 for sure

**Reply by pandeyanujk** [reply]
I think it's a good idea. Blogging using IPFS would be cool.
